torch/quantization/quantization_mappings.py

```diff
@@ -179,5 +179,5 @@ def get_default_special_act_post_process(module_cls):
     """
     return DEFAULT_MODULE_TO_ACT_POST_PROCESS.get(module_cls, None)
 
-def has_special_act_post_process(module_cls):
-    return module_cls in DEFAULT_MODULE_TO_ACT_POST_PROCESS
+def has_special_act_post_process(module):
+    return module.training and type(module) in DEFAULT_MODULE_TO_ACT_POST_PROCESS
```

Comment:
nit :)

I feel like in cases like this we should check if it is a class or an instance. You can do it using `inspect.isclass`. Here is the behavior:

```python
import inspect
class Foo:
  pass
f = Foo()
inspect.isclass(f)  # Returns "False"
inspect.isclass(Foo)  # Returns "True"
``` 

That way in case someone tries to pass a type, it will be handled

```python
if inspect.isclass(module):
  raise ValueError(f"Expecting an instance of a class, got {module}")
```

Reply:
this is probably too complicated for a small helper function :) but thanks for the suggestion. I'll change this to be an internal helper function.